rerun_py/rerun_bindings/rerun_bindings.pyi

```diff
@@ -1907,3 +1907,67 @@ class NotFoundError(Exception):
 
 class AlreadyExistsError(Exception):
     """Raised when trying to create a resource that already exists."""
+
+class ServerInternal:
+    """
+    Internal Rerun server instance.
+
+    This is the low-level binding to the Rust server implementation.
+    Users should typically use `rerun.server.Server` instead.
+    """
+
```

Comment:
FYI, you don't need to provide docstrings for `*Internal` objects here. They aren't checked like the others. I like this particular class level docstring, but I think i'd remove the method ones, because they will rot (since they are not checked).